grakn-graph/src/main/java/ai/grakn/graph/internal/ThingImpl.java

```diff
@@ -78,7 +78,7 @@
         }
 
         return vertex().graph().factory().buildConcept(typeEdge.get().target());
-    });
+    }, Cacheable.type());
```

Comment:
This is so picky, but you should make Cacheable the first parameter! For prettiness sake :P